### ISSUE TITLE
Not working on 7.1.12?

### ISSUE THREAD

**Issue by otto2704** [Weilbyte/PVEDiscordDark#128]
Hi!
I am on PVE 7.1.12 now and it seems that the dark mode does not work anymore. Tried installing online and offline, the script runs without any errors, but web interface stays white :-(
Maybe I did something wrong?

**Comment by Weilbyte** [comment]
Have you tried clearing browser cache?

**Comment by otto2704** [comment]
yes, even tried a different browser....

**Comment by Weilbyte** [comment]
Odd. Please share contents of `/usr/share/pve-manager/index.html.tpl`

**Comment by otto2704** [comment]
of course:
`<!DOCTYPE html>
<html>
  <head>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
    <title>[% nodename %] - Proxmox Virtual Environment</title>
    <link rel="icon" sizes="128x128" href="/pve2/images/logo-128.png" />
    <link rel="apple-touch-icon" sizes="128x128" href="/pve2/images/logo-128.png" />
    <link rel="stylesheet" type="text/css" href="/pve2/ext6/theme-crisp/resources/theme-crisp-all.css?ver=7.0.0" />
    <link rel="stylesheet" type="text/css" href="/pve2/ext6/crisp/resources/charts-all.css?ver=7.0.0" />
    <link rel="stylesheet" type="text/css" href="/pve2/fa/css/font-awesome.css" />
    <link rel="stylesheet" type="text/css" href="/pve2/css/ext6-pve.css?ver=[% version %]" />
    <link rel="stylesheet" type="text/css" href="/pwt/css/ext6-pmx.css?ver=[% wtversion %]" />
    [% IF langfile %]
    <script type='text/javascript' src='/pve2/locale/pve-lang-[% lang %].js?ver=[% version %]'></script>
    [%- ELSE %]
    <script type='text/javascript'>function gettext(buf) { return buf; }</script>
    [% END %]
    [%- IF debug %]
    <script type="text/javascript" src="/pve2/ext6/ext-all-debug.js?ver=7.0.0"></script>
    <script type="text/javascript" src="/pve2/ext6/charts-debug.js?ver=7.0.0"></script>
    [%- ELSE %]
    <script type="text/javascript" src="/pve2/ext6/ext-all.js?ver=7.0.0"></script>
    <script type="text/javascript" src="/pve2/ext6/charts.js?ver=7.0.0"></script>
    [% END %]
    <script type="text/javascript" src="/pve2/js/u2f-api.js"></script>
    <script type="text/javascript" src="/qrcode.min.js"></script>
    <script type="text/javascript">
    Proxmox = {
        Setup: { auth_cookie_name: 'PVEAuthCookie' },
        defaultLang: '[% lang %]',
        NodeName: '[% nodename %]',
        UserName: '[% username %]',
        CSRFPreventionToken: '[% token %]'
    };
    </script>
    <script type="text/javascript" src="/proxmoxlib.js?ver=[% wtversion %]"></script>
    <script type="text/javascript" src="/pve2/js/pvemanagerlib.js?ver=[% version %]"></script>
    <script type="text/javascript" src="/pve2/ext6/locale/locale-[% lang %].js?ver=7.0.0"></script>

    <script type="text/javascript">
    if (typeof(PVE) === 'undefined') PVE = {};
    Ext.History.fieldid = 'x-history-field';
    Ext.onReady(function() { Ext.create('PVE.StdWorkspace');});
    </script>

  </head>
  <body>
    
    <form id="history-form" class="x-hidden">
    <input type="hidden" id="x-history-field"/>
    </form>
  </body>
</html>`

**Comment by Weilbyte** [comment]
Right, it seems to be missing some stuff that the installer should have appended to it. Are you running the script as root?

**Comment by otto2704** [comment]
of course as root....
argh... idiotic me...
did not see, that the script also needs an install parameter :-(
without parameters the script just exits without any error...
maybe you could add something like this for the future (just in case that there are other idiots like me):
`parse_cli()
{
        if [ $# = 0 ]; then
                usage;
                exit 1;
        fi
        ....
`
Or should the script do something if called without parameters?
